### PR TITLE
add digitalsignalperson/obsidian-rofi-helper

### PR DESCRIPTION
**Pull request by digitalsignalperson**
# I am submitting a new Community Plugin

## Repo URL

Link to my plugin: https://github.com/digitalsignalperson/obsidian-rofi-helper

## Release Checklist
- [x] I have tested the plugin on
  - [ ]  Windows
  - [ ]  macOS
  - [x]  Linux
  - [ ]  Android _(if applicable)_
  - [ ]  iOS _(if applicable)_
- [x] My GitHub release contains all required files
  - [x] `main.js`
  - [x] `manifest.json`
  - [ ] `styles.css` _(optional)_
- [x] GitHub release name matches the exact version number specified in my manifest.json (_**Note:** Use the exact version number, don't include a prefix `v`_)
- [x] The `id` in my `manifest.json` matches the `id` in the `community-plugins.json` file.
- [x] My README.md describes the plugin's purpose and provides clear usage instructions.
- [x] I have read the tips in https://github.com/obsidianmd/obsidian-releases/blob/master/plugin-review.md and have self-reviewed my plugin to avoid these common pitfalls.
- [x] I have added a license in the LICENSE file.
- [x] My project respects and is compatible with the original license of any code from other plugins that I'm using.
      I have given proper attribution to these other projects in my `README.md`.

### CODE REVIEW

**Comment by joethei** [comment]
Hi @digitalsignalperson do you still intend on submitting this plugin?
Please fix the validation errors so we can review it.

File: community-plugins.json [hunk 1]
```diff
@@ -6872,6 +6872,12 @@
     "author": "Mo Ismat",
     "description": "Hides the cursor when scrolling and shows it again when moving the mouse.",
     "repo": "moismat/obsidian-auto-hide-cursor"
+  },
+  {
+    "id": "rofi-helper",
+    "name": "Rofi Helper Plugin",
+    "description": "This plugin adds a leaf id parameter to the URI protocol for switching between open obsidian tabs with Rofi. A sample Rofi script is included.",
```
Comment (liamcain) [inline community-plugins.json:6879]:
```suggestion
    "description": "Adds a leaf ID parameter to the URI protocol for switching between open obsidian tabs with Rofi. A sample Rofi script is included.",
```

File: community-plugins.json [hunk 2]
```diff
@@ -6872,6 +6872,12 @@
     "author": "Mo Ismat",
     "description": "Hides the cursor when scrolling and shows it again when moving the mouse.",
     "repo": "moismat/obsidian-auto-hide-cursor"
+  },
+  {
+    "id": "rofi-helper",
+    "name": "Rofi Helper Plugin",
```
Comment (liamcain) [inline community-plugins.json:6878]:
```suggestion
    "name": "Rofi Helper",
```

**Comment by liamcain** [comment]
Hi @digitalsignalperson, sorry for the delay in getting to reviewing this plugin. Plugin looks good to me. There's just a small issue on the validation. Do you mind updating the plugin name in the repo to match? We're trying to avoid plugins including the word "plugin" in their name for simplicity.

**Pull request merged**